### PR TITLE
Respect original data type of onboard_num

### LINKED ISSUE #122: onboard_num failing to parse alphanumeric assignments
`onboard_num` is a string in the mission file but is converted to an integer when interpreted by this module.

This causes the following exception when a mission is loaded that contains a unit with an alphanumerical tail number
>ValueError: invalid literal for int() with base 10: 'E3A'

Is this an intentional design choice?

### PR DESCRIPTION
**Pull request by ColinM9991**
Resolves #122 

Respects the original data type of onboard_num while still allowing uniquely generated tail numbers.

Ran all unit tests before and after making this change with no discrepancies.

I've only recently picked up Python after 7-8 years of years since I last touched it, please do tell me if there's something I haven't done or confirmed.

### CODE REVIEW

File: dcs/country.py [hunk 1]
```diff
@@ -140,18 +140,14 @@ def next_callsign_category(self, category):
             self.current_callsign_category[category] = 0
         return self.callsign.get(category)[self.current_callsign_category[category]]
 
-    @property
-    def unused_onboard_numbers(self) -> Set[int]:
```
Comment (rp-) [inline dcs/country.py:144]: any reason why this property was removed?

File: dcs/country.py [hunk 2]
```diff
@@ -161,8 +157,8 @@ def reserve_onboard_num(self, number: int) -> bool:
         self._tail_numbers.add(number)
         return is_in
 
-    def next_onboard_num(self) -> int:
-        free_set = {x for x in range(10, 999)} - self._tail_numbers
+    def next_onboard_num(self) -> str:
+        free_set = {str(x) for x in range(10, 999)} - self._tail_numbers
```
Comment (rp-) [inline dcs/country.py:161]: we should here atleast use the same format, as we did for serialization: `"{:03}"`

**Review by rp- — CHANGES_REQUESTED** [review]

File: dcs/country.py (hunk 1 above)
Comment (ColinM9991) [reply dcs/country.py:144]: I searched high and low for usages and couldn't find any. I can re-add of this is used by consumers.

Comment (rp-) [reply dcs/country.py:144]: well pydcs is a library and so there will be a lot of methods/functions that are potentially not used within the library

Comment (ColinM9991) [reply dcs/country.py:144]: Reintroduced this property

File: dcs/country.py (hunk 2 above)
Comment (ColinM9991) [reply dcs/country.py]: Format has been changed to what it was originally being serialized as

**Pull request merged**